src/components/ItaliaTheme/CustomerSatisfaction/FeedbackForm.jsx

```diff
@@ -214,7 +214,7 @@ const FeedbackForm = ({ contentType, pathname }) => {
       <Container>
         <Row className="d-flex justify-content-center bg-primary">
           <Col className="col-12 col-lg-6">
-            <div className="feedback-form">
+            <div className="feedback-form" role="alert">
```

Comment:
Cambierei solo role="form" al posto di "alert"

Reply:
c'era scritto così in questo ticket ma non so se è stato messo per errore https://redturtle.tpondemand.com/entity/47111-a11y-item-51-customer-satisfaction-radio